org.adempiere.base/src/org/compiere/wf/MWFNextCondition.java

```diff
@@ -114,15 +116,35 @@ public boolean isOr()
 	 *	@return true if true
 	 */
 	public boolean evaluate (MWFActivity activity)
+	{
+		return evaluate(activity.getPO());
+	}	//	evaluate
+	
+	/**
+	 * 	Evaluate Condition
+	 * 	@param po PO
+	 *	@return true if true
+	 */
+	public boolean evaluate (PO po)

```

Comment:
Is there any advantage adding this method?  I think is not necessary.

Reply:
It's used in my new plugin, which is based on this modifications.
Maybe I should commit this method separately.

Reply:
no problem - if you need it is OK, I was just wondering if it was necessary